toqito/matrix_props/is_absolutely_k_incoherent.py

```diff
@@ -0,0 +1,121 @@
+"""Checks if the matrix is absolutely $k$-incoherent."""
+
+import cvxpy as cp
+import numpy as np
+
+from toqito.matrix_props import is_positive_semidefinite, is_square
+
+
+def is_absolutely_k_incoherent(mat: np.ndarray, k: int, tol: float = 1e-15) -> bool:
+    r"""Determine whether a quantum state is absolutely k-incoherent :cite:`Johnston_2022_Absolutely`.
+
+    Formally, a mixed quantum state is said to be absolutely k-incoherent if :math:`U \rho U^* \in \mathbb{I}_{k, n}`
+    for all unitary matrices :math:`U \in \text{U}(\mathbb{C}^n)`.
```

Comment:
where $n$ is....

Reply:
Done in f6636dc808310c7901354bfc3082e63794cd0567